### PR TITLE
fix(ingestion/airflow-plugin): ingesting the tags along with the data

### PR DESCRIPTION
**Pull request by dushayntAW**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/client/airflow_generator.py [hunk 1]
```diff
@@ -18,9 +18,9 @@
 assert AIRFLOW_PATCHED
 
 if TYPE_CHECKING:
-    from airflow import DAG
     from airflow.models import DagRun, TaskInstance
 
+    from airflow import DAG
     from datahub_airflow_plugin._airflow_shims import Operator
```
Comment (anshbansal) [inline metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/client/airflow_generator.py:24]: Please revert unnecessary change

File: metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/datahub_listener.py [hunk 2]
```diff
@@ -5,23 +5,25 @@
 import threading
 from typing import TYPE_CHECKING, Callable, Dict, List, Optional, TypeVar, cast
 
-import airflow
```
Comment (anshbansal) [inline metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/datahub_listener.py:8]: Have you run lint on this PR?

Comment (dushayntAW) [reply metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/datahub_listener.py:8]: yes.. I executed lint and as a part of that only this files were modified

File: metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/client/airflow_generator.py (hunk 1 above)
Comment (dushayntAW) [reply metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/client/airflow_generator.py:24]: I have executed `../gradlew :metadata-ingestion-modules:airflow-plugin:lintFix` and it has edited these files

**Comment by anshbansal** [comment]
Airflow related CI passed. Others are unrelated
